src/System.Memory/src/System.Memory.csproj

```diff
@@ -27,12 +24,22 @@
     <Compile Include="System\SpanHelpers.Clear.cs" />
     <Compile Include="System\ThrowHelper.cs" />
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true'">
```

Comment:
Since it looks like you're also enabling CoreCLR's span with this change, could you please also remove the restriction in Configuration.props that builds for Windows only? This would work fine I think:
```xml
    <BuildConfigurations>
      netcoreapp;
    </BuildConfigurations>
```

And can remove any windows-specific stuff in the csproj.